source/Hardpoint.cpp

```diff
@@ -191,7 +192,24 @@ bool Hardpoint::CanAim(const Ship &ship) const
 // Check if this weapon is ready to fire.
 bool Hardpoint::IsReady() const
 {
-	return outfit && burstReload <= 0. && burstCount;
+	return outfit && burstReload <= 0. && burstCount && !IsBlind();
+}
+
+
+
+// Check if this weapon can't fire because of its blindspots.
+bool Hardpoint::IsBlind() const
+{
+	return any_of(baseAttributes.blindspots.begin(), baseAttributes.blindspots.end(),
+		[this](pair<Angle, Angle> blindspot)
+		{
+			if(blindspot.first.Degrees() < blindspot.second.Degrees())
```

Comment:
```suggestion
			if(blindspot.first.Degrees() <= blindspot.second.Degrees())
```
If min and max are exactly the same, then we we should not have a blindspot of 360 degrees, but only a blindspot if we have an exact match.